### PR TITLE
Added extensible buttons next to dialog close button

### PR DESCRIPTION
**Pull request by rickyp-ms**
#### Pull request checklist

- [x] Addresses an existing issue: #838
- [x] Include a change request file if publishing 
- [x] New feature, bugfix, or enhancement
  - [x] Includes tests
- [x] Documentation update

#### Description of changes

Adding icons next to the close button in the dialog.

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/Dialog/Dialog.tsx [hunk 1]
```diff
@@ -130,6 +131,10 @@ export class Dialog extends BaseComponent<IDialogProps, IDialogState> {
                 <div className='ms-Dialog-header'>
                   <p className='ms-Dialog-title' id={ id + '-title' }>{ title }</p>
                   <div className='ms-Dialog-topButton'>
+                    { this.props.topButtonsProps.map((props) => (
+                      <Button {...props}>
+                      </Button>
```
Comment (mdahamiwal) [inline packages/office-ui-fabric-react/src/components/Dialog/Dialog.tsx]: minor: end in one line `<Button {...props} />`

Comment (rickyp-ms) [inline packages/office-ui-fabric-react/src/components/Dialog/Dialog.tsx]: Closed in latest review

**Comment by rickyp-ms** [comment]
I don't see any active comments... can I get a sign off?

**Comment by rickyp-ms** [comment]
Adding dzearing to get a sign off...

**Comment by rickyp-ms** [comment]
Hello???

**Comment by rickyp-ms** [comment]
@mdahamiwal Can I get a sign off?

**Comment by rickyp-ms** [comment]
@dzearing Anyway you maybe able to give me a quick sign off here or at least a comment as to why not? We're looking to ship in 2 months and this is a blocking work item for us.

**Pull request merged**